CHANGES

```diff
@@ -1,7 +1,7 @@
 Bleach changes
 ==============
 
-Version 3.0.3 (In development)
+Version 3.1.0 (In development)
```

Comment:
We're adding backwards-compatible API changes, so this needs to be a MINOR increment rather than a PATCH increment.